Added description to move databases between k8s instances (or namespaces) (#165)
konveyor/tackle-controls: docs/MovingTackleDBs.md

The [Tackle Operator](https://operatorhub.io/operator/tackle-operator) installs a set of tackle-* component and database pods, which are working together as the "Tackle Application Assessment" - for historical reasons also referred to as "Pathfinder".

![topology](./images/tackle-topology-view.png)

To move everything from one Kubernetes instance to another, we need to dump/import the contents of the postgresql DBs, with the exception of the tackle-keycloak-postgresdb database pod.

So, make sure you have the Tackle Operator installed in the target system as well and have it up and running.

**THIS WILL OVERWRITE EVERYTHING YOU HAVE IN YOUR DESTINATION TACKLE PODS**

**NOTE** I'm using `oc` here instead of `kubectl` since this is running on a local Red Hat CRC (Code Ready Containers) installation, but you can replace `oc` with `kubctl` - it just happens that this was "hacked" on CRC, not minishift and I didn't have a `kubectl` CLI pointing at my CRC ready.

## Steps to take

### 1) On the source system (or namespace), get your postgres pods (ignore the keycloak-postgresql)

```

> [🎩︎mnagel konveyor]$ oc get pods | grep postgres

Reviewer: ```suggestion
$ oc get pods | grep postgres
```

tackle-application-inventory-postgresql-5f959df4df-zg46k   1/1     Running   0          15d
tackle-controls-postgresql-6bfd5d57ff-cbnfr                1/1     Running   0          15d
tackle-keycloak-postgresql-9cf6f6879-gbx5w                 1/1     Running   0          15d
tackle-pathfinder-postgresql-5557cb8bff-vgzng              1/1     Running   0          15d

> [🎩︎mnagel konveyor]$

Reviewer: ```suggestion
$
```

```

### 2) (repeat for every pod) get the DB user

```
oc exec tackle-application-inventory-postgresql-5f959df4df-zg46k -- printenv POSTGRES_USER
```

### 3) Export (dump) the database without the flyway tables. 

> As opposed to the development instructions in the [README.md](../README.md#db-mgmt), this uses the `--clean --if-exists` flags, and therefore needs to drop the `-a` flag (data only).

Reviewer: ```suggestion
As opposed to the development instructions in the [README.md](../README.md#database-management), this uses the `--clean --if-exists` flags, and therefore needs to drop the `-a` flag (data only).
```
